ocaml/typing/outcometree.mli

```diff
@@ -111,7 +101,31 @@ type out_ret_mode =
   (** The ret type is arrow, and need to print parens around the arrow, with
       modes annotating. *)
 
-type out_type =
+(** Represents a user-written jkind annotation *)
+type out_jkind_user =
```

Comment:
I think these should be folded in with `out_jkind`.

Reply:
Fixed